packages/core/src/runtime/stylis/prefixerPlugin.ts

```diff
@@ -34,11 +34,16 @@ export function prefix(value: string, length: number, children?: Element[]): str
     case 4599:
     case 4855:
       return WEBKIT + value + value;
-    // background-clip: text
+    // backdrop-filter, background-clip: text
     case 4215:
-      switch (charat(value, length + 1)) {
-        case 116:
-          return WEBKIT + value + value;
+      // Additional check on "backdrop-(f)ilter" to prevent conflict with "background-clip"
```

Comment:
```suggestion
      // Additional check on "backdrop-(f)ilter" to prevent conflict with "background-clip"
      // https://github.com/thysultan/stylis/issues/292
```